lib/salus/repo.rb

```diff
@@ -95,5 +95,18 @@ class Repo
     def initialize(path_to_repo = nil)
       @path_to_repo = path_to_repo
     end
+
+    ##
+    # @param [Array] args CLI rg command and options to run
+    # @return [Array<String>] Relative path of files returned
+    # from running the command.
+    def run_rg(*args)
+      data = IO.popen(args, chdir: @path_to_repo).read
+      return [] if data == ""
+
+      files = data.lines.map { |file| File.join(path_to_repo, file.strip) }
```

Comment:
Do we need to create absolute path? Instead of this, can we use the relative path?

`files = data.lines.map(&:strip)`

Reply:
I was getting unit test errors when I used a relative path

Reply:
Was the original code outputting absolute paths or relative paths?

Reply:
It returned the absolute path

Reply:
Example of an error when using relative path
``` 
 3) Salus::Scanners::Bandit#should_run? setup.cfg present but no requirements.txt should return true
     Failure/Error: expect(repo.py_files_present?).to eq([py_file])

       expected: ["spec/fixtures/python/python_project_no_req_txt/main.py"]
            got: ["main.py"]
```